pallets/did/src/benchmarking.rs

```diff
@@ -148,71 +160,100 @@ benchmarks! {
-	// 	/* code to test the function benchmarked */
-	// }
-	// verify {
-	// 	/* optional verification */
-	// 	assert_eq!(true, true)
-	// }
+	// ---------------------------------------------
+	revoke_issuer {
+		let issuer = issuer::<T>(1);
+		let info = IssuerInfo { status: IssuerStatus::Revoked };
+		DID::<T>::add_issuer(RawOrigin::Root.into(), issuer.clone());
+	}: _(
+		RawOrigin::Root, issuer.clone()
+	)
+	verify {
+		assert_eq!(Issuers::<T>::get(issuer), Some(info));
+	}
 
-	// // ---------------------------------------------
-	// reactivate_issuer {
-	// 	/* code to set the initial state */
-	// }: {
-	// 	/* code to test the function benchmarked */
-	// }
-	// verify {
-	// 	/* optional verification */
-	// 	assert_eq!(true, true)
-	// }
+	// ---------------------------------------------
+	reactivate_issuer {
+		let issuer = issuer::<T>(1);
+		let info = IssuerInfo { status: IssuerStatus::Active };
+		DID::<T>::add_issuer(RawOrigin::Root.into(), issuer.clone());
+		DID::<T>::revoke_issuer(RawOrigin::Root.into(), issuer.clone());
+	}: _(
+		RawOrigin::Root, issuer.clone()
+	)
+	verify {
+		assert_eq!(Issuers::<T>::get(issuer), Some(info));
+	}
 
-	// // ---------------------------------------------
-	// remove_issuer {
-	// 	/* code to set the initial state */
-	// }: {
-	// 	/* code to test the function benchmarked */
-	// }
-	// verify {
-	// 	/* optional verification */
-	// 	assert_eq!(true, true)
-	// }
+	// ---------------------------------------------
+	remove_issuer {
+		let issuer = issuer::<T>(1);
+		DID::<T>::add_issuer(RawOrigin::Root.into(), issuer.clone());
+		DID::<T>::revoke_issuer(RawOrigin::Root.into(), issuer.clone());
+	}: _(
+		RawOrigin::Root, issuer.clone()
+	)
+	verify {
+		assert_eq!(Issuers::<T>::get(issuer), None);
+	}
 
-	// // ---------------------------------------------
-	// add_credentials_type {
-	// 	/* code to set the initial state */
-	// }: {
-	// 	/* code to test the function benchmarked */
-	// }
-	// verify {
-	// 	/* optional verification */
-	// 	assert_eq!(true, true)
-	// }
+	// ---------------------------------------------
+	add_credentials_type {
+		let m in 0 .. T::MaxCredentialsTypes::get();
+		let mut credentials_types = BoundedVec::default();
+		CredentialsTypes::<T>::put(credentials_types.clone());
 
-	// // ---------------------------------------------
-	// remove_credentials_type {
-	// 	/* code to set the initial state */
-	// }: {
-	// 	/* code to test the function benchmarked */
-	// }
-	// verify {
-	// 	/* optional verification */
-	// 	assert_eq!(true, true)
-	// }
+		for i in 0 .. m {
+			let mut credentials_types:
+				frame_support::BoundedVec<
+					frame_support::BoundedVec<u8, T::MaxString>,
+					T::MaxCredentialsTypes
+				> = BoundedVec::default();
+			let cred = create_credential_type::<T>(i as u8);
+			credentials_types.try_push(cred.clone());
+		}
+
+		CredentialsTypes::<T>::put(credentials_types.clone());
+	}: _(
+		RawOrigin::Root, credentials_types.to_vec().clone()
```

Comment:
```suggestion
		RawOrigin::Root, credentials_types.clone()
```
Not necessary if `credential_types` is changed to a `Vec`